packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/update-transaction.spec.js

```diff
@@ -76,79 +74,96 @@ describe('remove-licence > update transaction', () => {
     jest.clearAllMocks()
   })
 
-  it('status.setCurrentPermission is being called with different permission', async () => {
-    const setPermission = getStatusPermission()
-    const mockRequest = createMockRequest({
-      cache: { status: { permissions: [getStatusPermission(), setPermission, getStatusPermission({ [REMOVE_LICENCE.page]: true })] } }
-    })
-    await updateTransaction(mockRequest)
-    expect(mockRequest.cache().helpers.status.setCurrentPermission()).toEqual(expect.objectContaining(setPermission))
-  })
-
-  it('transaction.set is being called with a permission removed', async () => {
+  it('transaction cache is updated after permission is deleted', async () => {
     const transaction = { permissions: [getTransactionPermissionOne(), getTransactionPermissionTwo(), getTransactionPermissionThree()] }
-    const mockRequest = createMockRequest({ cache: { transaction: transaction } })
+    const mockRequest = getMockRequest({ cache: { transaction: transaction } })
```

Comment:
1) You're using the default value for transaction - change it so it has a different order
2) You don't have to do `getMockRequest({ cache: { transaction: transaction } })`, you can do `getMockRequest({ cache: { transaction } })`